test/cypress/features/login.feature

```diff
@@ -18,55 +18,14 @@ Feature: Login
 
   Scenario: Log in to Testnet
     Given I am on Login page
-    When I click on settingsMenu
-    And I click on switchNetworksTrigger
-    And I click on closeDialog
     When I choose testnet
     When I enter first passphrase of testnet_guy
     When I login
     Then I should be connected to testnet
-    And I click on btcToken
-    And I click on logoutBtn
-    Then I should see lisk monitor features
 
   Scenario: Log in on devnet
     Given I am on Login page
-    When I click on settingsMenu
-    And I click on switchNetworksTrigger
-    And I click on closeDialog
     When I choose devnet
     When I enter first passphrase of genesis
     When I login
     Then I should be connected to devnet
-
-  Scenario: Log in to Mainnet (Network switcher is not enabled)
```

Comment:
Why is this removed?

Reply:
There were two tests testing the same thing.